src/Resources/skeleton/Admin.php.twig

```diff
@@ -16,14 +18,14 @@ use Sonata\AdminBundle\Show\ShowMapper;
 
 class {{ classBasename }} extends AbstractAdmin
 {
-    protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+    protected function configureDatagridFilters(DatagridMapper $datagridMapper): void
```

Comment:
But in the interface there is no void isn’t it?

Reply:
Yes. So what?

Reply:
I mean if the parent method in AbstractAdmin has no void return type, can we add it here? Isn’t it complaining about method signature differs? Currently on a phone and can’t check by myself

Reply:
The signature does not have to be the same, it only has to be compatible. Here is some proof of this: https://3v4l.org/G6NqK

Reply:
Great thank you 😊